shaazk/Free-Mentors
# #168234575 update files with eslint configurations

#### What does this PR do?
- it updates my files with ESlint configurations
#### Description of Task to be completed?
- my file codes should be well indented and follow the proper code structures
#### How should this be manually tested?
 -clone free mentors on your computer and open the older in your VS Code or any other html IDE. in your terminal, configure eslint and check all server files . there will not be any eslint error notifications and outlinings
#### What are the relevant pivotal tracker stories?
- [#168234575](https://www.pivotaltracker.com/story/show/168234575)
